styles/start.module.css

```diff
@@ -55,12 +55,12 @@
     width: 224px;
```

Comment:
Can we decrease the initial width so we can keep the animation of the prompt stretching out and turn into the board?